frontend/server/src/Controllers/Course.php

```diff
@@ -1324,6 +1324,139 @@ public static function schoolsIndexForSmarty(\OmegaUp\Request $r) {
         ];
     }
 
+    /**
+     * @return array{users: list<array{accepted: bool|null, admin?: array{name?: null|string, user_id?: int|null, username?: null|string}, country: null|string, country_id: null|string, course_id: int, identity_id: int, last_update: null|string, request_time: string, user_id: int|null, username: string}>}
+     */
+    public static function apiRequests(\OmegaUp\Request $r): array {
+        // Authenticate request
+        $r->ensureMainUserIdentity();
+
+        \OmegaUp\Validators::validateStringNonEmpty(
+            $r['course_alias'],
+            'course_alias'
+        );
+
+        $course = self::validateCourseExists($r['course_alias']);
+        if (is_null($course->course_id)) {
+            throw new \OmegaUp\Exceptions\NotFoundException('courseNotFound');
+        }
+        if (!\OmegaUp\Authorization::isCourseAdmin($r->identity, $course)) {
+            throw new \OmegaUp\Exceptions\ForbiddenAccessException();
+        }
+
+        $resultAdmins =
```

Comment:
```suggestion
        $resultAdmin =
```

esto sólo regresa un admin.

Reply:
Ya rediseñé la consulta de tal manera que en una sola se trajera la información del primer admin, así que esta variable se llama ahora `$resultRequestsForCourseWithFirstAdmin`